test/parallel/test-console.js

```diff
@@ -8,10 +8,6 @@ assert.ok(process.stderr.writable);
 assert.equal('number', typeof process.stdout.fd);
 assert.equal('number', typeof process.stderr.fd);
 
-assert.throws(function() {
```

Comment:
I didn't mean to delete the whole test. Just the one line that I commented on was unnecessary.